web-app/src/components/CreateUser/index.jsx

```diff
@@ -1,56 +1,158 @@
+      >
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'input',
+            name: 'forename',
+            placeholder: 'Enter a forename',
+          }}
+          value={formData.forename}
+          focus
+          label="Forename:"
+          rules={{
+            required: true,
+          }}
+        />
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'input',
+            name: 'surname',
+            placeholder: 'Enter a surname',
+          }}
+          value={formData.surname}
+          focus={false}
+          label="Surname:"
+          rules={{
+            required: true,
+          }}
+        />
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'email',
+            name: 'email',
+            placeholder: 'Enter an email',
+          }}
+          value={formData.email}
+          focus={false}
+          label="Email:"
+          rules={{
+            required: true,
+            isEmail: true,
+          }}
+        />
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'password',
+            name: 'password',
+            placeholder: 'Enter an password',
+          }}
+          value={formData.password}
+          focus={false}
+          label="Password:"
+          rules={{
+            required: true,
+            minLength: 6,
+          }}
+        />
+        <Input
+          type="select"
+          htmlAttrs={{
+            name: 'country',
+            options: [
+              { value: 1, displayValue: 'Northern Ireland' },
+              { value: 2, displayValue: 'Mexico' },
+            ],
+          }}
+          value={formData.country}
+          focus={false}
+          label="Country:"
+        />
+        <Input
+          type="select"
+          htmlAttrs={{
+            name: 'status',
+            options: [
+              { value: employeeStatus.ACTIVE, displayValue: 'Active' },
+              { value: employeeStatus.INACTIVE, displayValue: 'Inactive' },
+            ],
+          }}
+          value={formData.status}
+          focus={false}
+          label="Status:"
+        />
+        <Input
+          type="select"
+          htmlAttrs={{
+            name: 'employeeRole',
+            options: [
+              { value: 1, displayValue: 'Team Leader' },
+              { value: 2, displayValue: 'System Admin' },
+              { value: 3, displayValue: 'Employee' },
+            ],
+          }}
+          value={formData.employeeRole}
+          focus={false}
+          label="Employee Role:"
+        />
+        <Input
+          type="date"
+          htmlAttrs={{
+            type: 'input',
+            name: 'startDate',
+            placeholder: 'Enter a start date',
+          }}
+          value={formData.startDate}
+          rules={{
+            dateNotInPast: true,
+          }}
+          focus={false}
+          label="Start Date:"
+        />
```

Comment:
yeah I feel these inputs could be wrapped up in another file and referred to here

Reply:
I totally agree and will do this in a much smaller PR once I get this approved.